<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarehouseOwnerMenu</class>
 <widget class="QMainWindow" name="WarehouseOwnerMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="ownerFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>371</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="emailInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Компания или ИП</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Контактный телефон</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Контактная почта</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="taxNumberInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ИНН</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameInput">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Информация о пользователе</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="warehouseInfoFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="addressInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Информация о складе</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Вместимость</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CapacityInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QComboBox" name="warehouseSelection">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="infoTypeWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>651</width>
      <height>231</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="employeePage">
     <widget class="QTableWidget" name="employeeTable">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>611</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="switchToProduct">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>К продуктам</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="productPage">
     <widget class="QTableWidget" name="productTable">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>611</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="switchToEmployee">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>К сотрудникам</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
